### PR TITLE
Introduce form-js

### LINKED ISSUE #87: Integrate form-js into the site
__What should we do?__

* [x] Add form-js to the site
* [ ] Blog about form-js being there now

__Why should we do it?__

form-js is a great addition to our tools, will soon be ready to use and should be marketed accordingly.

### PR DESCRIPTION
**Pull request by nikku**
Closes #87

### CODE REVIEW

File: src/pages/toolkit/form-js/index.hbs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  </div>
+</div>
+
+<!-- features -->
+
+<div class="bi-features">
+  <div class="container">
+    <h2>Create forms visually</h2>
+    <p class="lead">
+      Assemble your forms using our intuitive <a href="https://github.com/bpmn-io/form-js#builder">form builder</a>.
```
Comment (pinussilvestrus) [inline src/pages/toolkit/form-js/index.hbs:35]: Is it rather this URL: https://github.com/bpmn-io/form-js#create-and-edit-a-form? Or will we update them?

**Review by pinussilvestrus — COMMENTED** [review]
Nice stuff 👍 
Will we update the home page as well?

image

File: src/pages/toolkit/form-js/index.hbs (hunk 1 above)
Comment (nikku) [reply src/pages/toolkit/form-js/index.hbs:35]: Good catch. I believe we're better of adding stable in-page links. Created https://github.com/bpmn-io/form-js/pull/64 for `#viewer` and `#builder` to work.

**Comment by nikku** [comment]
Good catch on the landing page. Will remove cmmn-js there, too and add a form-js screen capture.

**Comment by nikku** [comment]
Integrated form-js into landing page via https://github.com/bpmn-io/bpmn.io/pull/88/commits/4bc144720bc0b37788ab9e589f4c28f0fbc09131.

**Comment by barmac** [comment]
https://github.com/bpmn-io/bpmn.io/pull/88#issuecomment-847607621

We did not, but will handle this via https://github.com/bpmn-io/bpmn.io/issues/222 :)

**Pull request merged**